installing/installing_two_node_cluster/installing_tnf/install-tnf.adoc

```diff
@@ -0,0 +1,122 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="installing-tnf_{context}"]
+= Installing a two-node OpenShift cluster with fencing
+include::_attributes/common-attributes.adoc[]
+:context: install-tnf
+
+toc::[]
+
+You can deploy a two-node OpenShift cluster with fencing by using either the installer-provisioned infrastructure (IPI) or the user-provisioned infrastructure (UPI) installation method. The following examples provide sample `install-config.yaml` configurations for both methods.
+
+[NOTE]
+====
+It is recommended to take an etcd backup before proceeding to ensure that you can restore the cluster if any issues occur.
+====
+
+[id="sample-install-config-two-node-fencing-ipi_{context}"]
```

Comment:
This section should be in a separate module, especially with DITA migration on the horizon.